public/index.html

```diff
@@ -2,42 +2,10 @@
 <html lang="en">
```

Comment:
한글 서비스이니 ko 로 해주면 더 좋겠네요! 

```suggestion
<html lang="ko">
```